app/src/main/res/layout/fragment_pill_box_management.xml

```diff
@@ -6,7 +6,7 @@
 
     <GridLayout
         android:layout_width="match_parent"
-        android:layout_height="0dp"
```

Comment:
You can set view width and height to `1/3` screen width with following code:
```
int width = Resources.getSystem().getDisplayMetrics().widthPixels;
LinearLayout gridItem = itemView.findViewById(R.id.grid_item);
ViewGroup.LayoutParams layoutParams = gridItem.getLayoutParams();
layoutParams.height = width / 3;
layoutParams.width = width / 3;
gridItem.requestLayout();
```

Reply:
Roger, thanks